lib/logstash/outputs/elasticsearch/default-ilm-policy.json

```diff
@@ -0,0 +1,14 @@
+{
+  "policy" : {
+    "phases": {
+      "hot" : {
+        "actions" : {
+          "rollover" : {
+            "max_size" : "50gb",
+            "max_age":"30d"
```

Comment:
we should also include in the policy that the indices that are created will have the current date - the creation date for the index - to give an idea of what kind of contents exist in the index

Reply:
@jsvd The default index pattern now includes the date - is this what you were referring to?

Reply:
right, all good 👍